cucumber/api/features/authenticate.feature

```diff
@@ -21,6 +21,19 @@ Feature: Exchange a role's API key for a signed authentication token
       cucumber:user:alice successfully authenticated with authenticator authn
     """
 
+  Scenario: Authenticate response should be encoded if asked
+    Then I can POST "/authn/cucumber/alice/authenticate?encoded=true" with plain text body ":cucumber:user:alice_api_key"
+    And the HTTP response content type is "text/plain"
```

Comment:
Then

Reply:
ding
I agree. But I used to main authentication test above as an example.. Should I change it too?